newdle/client/src/client.js

```diff
@@ -32,15 +33,33 @@ class Client {
     return this._request(flask`api.users`({q}));
   }
 
-  async _request(url, withStatus = false, isRetry = false) {
-    const headers = {Accept: 'application/json'};
+  createNewdle() {
+    return this._request(flask`api.create_newdle`(), {
+      method: 'POST',
+      body: JSON.stringify({
+        title: getTitle(this.store.getState()),
+        duration: getDuration(this.store.getState()),
+        timezone: 'UTC',
+        time_slots: {
+          start: moment().format('YYYY-MM-DDThh:mm'),
+          end: moment()
+            .add(15, 'minutes')
+            .format('YYYY-MM-DDThh:mm'),
+        },
+      }),
+    });
+  }
+
+  async _request(url, options = {}, withStatus = false, isRetry = false) {
+    const headers = {Accept: 'application/json', ...options};
```

Comment:
why are you unpacking options into headers?

Reply:
Somehow rebased the wrong code. It shouldn't be here. I'll fix it.